packages/@react-spectrum/searchfield/src/SearchField.tsx

```diff
@@ -30,7 +30,7 @@ function SearchField(props: SpectrumSearchFieldProps, ref: RefObject<TextFieldRe
       UNSAFE_className={
         classNames(
           styles,
-          'spectrum-Search-icon'
+          'spectrum-Textfield-icon'
```

Comment:
I don't believe this is needed, it is not a class in `import styles from '@adobe/spectrum-css-temp/components/search/vars.css';` (or really the two files it imports)
If it is, we should name it differently

this class is added in TextFieldBase from Textfield css https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/textfield/src/TextFieldBase.tsx#L92

Reply:
🤦, @snowystinger is correct here, you can go ahead and remove the entire UNSAFE_className block here

Reply:
Nice, thank you for checking!